src/components/Book.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+
+class Book extends React.Component {
+  render() {
+    const book = this.props.book;
```

Comment:
Destructuring the book props in this case should be `const { book} = this.props.book;`